src/components/UserSelector.tsx

```diff
@@ -73,7 +76,7 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                dispatch(authorSlice.actions.setAuthor(user));
               }}
```

Comment:
Using a link (`<a>` element) and manipulating the `href` attribute for handling click events is not a recommended practice in React. This could lead to unexpected behavior and is not accessible. Instead, consider using a `<button>` element for click events.